kubevirt/step1.md

```diff
@@ -0,0 +1,51 @@
+### Wait for the Kubernetes cluster to be ready
+
+Before we can start, we need to wait for the Kubernetes cluster to be ready (command prompt will appear to accept commands).
+
+#### Deploy KubeVirt
+
+Deploy KubeVirt operator[^1] using latest KubeVirt version (we query GH api to get latest release available).
+
+[^1]: An Operator is a method for packaging, deploying and managing Kubernetes applications, read more at [CoreOS Operators](https://coreos.com/operators/)
+
+`export KUBEVIRT_VERSION=$(curl -s https://api.github.com/repos/kubevirt/kubevirt/releases/latest|jq '.tag_name'|tr -d '"')
```

Comment:
Even slightly simpler with `jq`: no need to post-process wit `tr`, just use `-r`:

`export KUBEVIRT_VERSION=$(curl -s https://api.github.com/repos/kubevirt/kubevirt/releases/latest | jq 
 -r .tag_name)`

Reply:
updated to it